tests/browser/heda_settings_affiliations_edit_mode.robot

```diff
@@ -0,0 +1,51 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Go To Heda Home
+    Go To Heda Home
+    Select Frame			//iframe[contains(@id, "vfFrameId")]
+    Click Link				//a[contains(text(),'Affiliations')]
+    Wait Until Element Is visible	//a[contains(text(),'Affiliations')]
+    Capture Page Screenshot
+
+Validate Edit Mode For Affiliations, Settings
+    
+    #Yes radio button
+    Go To Heda Home
+    Select Frame                    //iframe[contains(@id, "vfFrameId")]
+     #Select the 'Yes' radio button
+    Click Element                   //div[@data-aura-rendered-by='106:0']//input[@type='radio']
```

Comment:
This locator, and most of the others in this test, are using the aura id which isn't stable.  It gets defined on every page load.  Is there another way to create this locator without using the global-id from aura?

Reply:
The HEDA.py library file could define the keyword Get HEDA Locator to allow locator construction to live in the HEDA.py file.  Here's the method definition from the base Salesforce library for reference:
https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/robotframework/Salesforce.py#L205-L213

This would allow us to remove all locators from the test cases themselves and instead treat them as part of the library for interacting with HEDA.

Reply:
Resolved the aura errors.  Now using stable locator.